configure.ac

```diff
@@ -626,7 +633,7 @@ AC_CHECK_ALIGNOF([long])
 
 AS_IF([! $arch64],
   [AS_IF([test x"$ac_cv_alignof_double" = "x8" ],
-    [AS_IF([ "$target_cpu" != "i386" ],
+    [AS_IF([test "$target_cpu" != "i386" ],
```

Comment:
Typo in your patch commit earlier - but shouldn't it be `i686` not `i386`?

Reply:
No because `i686` is arch64 (see line 634) so aligning doubles is irrelevant.
It is indeed only on i386 that the C compiler wants to align doubles even if the CPU doesn't require it.

Reply:
I don't follow? Regardless, though I haven't yet checked, doesn't this mean that msvc32 and mingw32 **do** get `ARCH_ALIGN_DOUBLE` where they didn't in `m-nt.h`?